include/libm2k/contextbuilder.hpp

```diff
@@ -116,6 +129,15 @@ LIBM2K_API Context* deviceOpen(const char* uri)
 }
 
 
+/**
+ * @private
+ */
+LIBM2K_API Context* deviceOpen(struct iio_context *ctx)
+{
+	return ContextBuilder::deviceOpen(ctx);
+}
+
+
```

Comment:
nitpick: extra line

Reply:
The lines were added in order to be style-consistent.